.gitignore

```diff
@@ -4,3 +4,6 @@
 src/*.o
 src/*.so
 src/*.dll
+/lib/
+/boost_1_57_0/
+/QuantLib-1.4.1/
```

Comment:
These two directories should not part of the repo / directory so we should not need gitignore.